mhr_api/report-templates/template-parts/registration/owners.html

```diff
@@ -56,8 +56,10 @@
                             {% if party.individualName.middle is defined %}&nbsp;{{ party.individualName.middle }}{% endif %}
                         {% endif %}
                     </div>
-                    {% if party.suffix is defined and party.suffix != '' %}
-                    <div class="pt-2">{{ party.suffix|upper }}</div>
+                    {% if party.description is defined and party.description != '' %}
```

Comment:
`description` field will have the same data in `suffix` for transfers (i saw in one of the PRs) so you got that covered with if/else condition. Just not sure what's the difference.

Reply:
Yes, the suffix is retained for report generation of legacy registrations; description is for new registrations and post data migration.

Reply:
Great, thanks for the clarification.